impeller/renderer/blit_pass.h

```diff
@@ -96,6 +101,8 @@ class BlitPass {
   ///                                 command.
   /// @param[in]  slice               For cubemap textures, the slice to write
   ///                                 data to.
+  /// @param[in]  convert_to_read     Whether to convert the texture to a shader
```

Comment:
I suppose we can convert this to a `UsageHints` mask if we need to add more. How about `set_read_usage_hint` or something similar to indicate this is advisory?

Reply:
The odd part is that this isn't advisory for Vulkan, you'll hit an error state if you bind the dest texture without this (assuming you opted out of shader read conversion).

We need a better system for determining layout transitions but I'm not sure what this is right now. Usage hints doesn't feel like the right approach since its a union of possible states and not the current state.

Reply:
You mentioned tracking the layouts at the queue level per image once. That seems like a good idea. Have we documented it in an issue?